spec/requests/v1/pacbio/libraries_spec.rb

```diff
@@ -23,38 +27,52 @@
       expect(json['data'][0]['attributes']['concentration']).to eq(library1.concentration)
       expect(json['data'][0]['attributes']['library_kit_barcode']).to eq(library1.library_kit_barcode)
       expect(json['data'][0]['attributes']['fragment_size']).to eq(library1.fragment_size)
-      expect(json['data'][0]["attributes"]["pacbio_tag_id"]).to eq(library1.pacbio_tag_id)
-      expect(json['data'][0]["attributes"]["pacbio_request_id"]).to eq(library1.pacbio_request_id)
+
+      requests = json['data'][0]['relationships']['requests']
+      expect(requests.length).to eq(1)
+      expect(requests['data'][0]['id'].to_s).to eq(request_library.id.to_s)
 
       expect(json['data'][1]['attributes']['volume']).to eq(library2.volume)
       expect(json['data'][1]['attributes']['concentration']).to eq(library2.concentration)
       expect(json['data'][1]['attributes']['library_kit_barcode']).to eq(library2.library_kit_barcode)
       expect(json['data'][1]['attributes']['fragment_size']).to eq(library2.fragment_size)
-      expect(json['data'][1]["attributes"]["pacbio_tag_id"]).to eq(library2.pacbio_tag_id)
-      expect(json['data'][1]["attributes"]["pacbio_request_id"]).to eq(library2.pacbio_request_id)
     end
 
   end
 
```

Comment:
we could add a test for update too

Reply:
Agreed. As discussed we will add this as an enhancement issue.